### ISSUE TITLE
delete

### ISSUE THREAD

**Issue by Hogerys** [TeamNewPipe/NewPipe#6263]

**Comment by mhmdanas** [comment]
Sorry, I'm not clear on what the incorrect behavior is here. Can you please clarify?

**Comment by SameenAhnaf** [comment]
@mhmdanas Hogerys prefers to postpone auto-queue function or showing related stream if a playlist or some videos are manually enqueued. Because, the user has already decided which videos to be enqueued. That's why, he prefers to see next videos on the enqueued list or queue manager in the section of related streams.

Yes, there's a 'Queue manager' icon on main player. But keeping both "Auto-queue" and "Play all from playlist" enabled makes no real sense. The user is not supposed to get distractions. He or she has already decided what to do with my videos.

I suggested a solution in https://github.com/TeamNewPipe/NewPipe/issues/6275#issuecomment-841669202 which may possibly solve both issues.

**Comment by Delikt** [comment]
i read this issue and the issue/solution in #6275 which have in my opinion nothing to do with each other...

the opener means  he need to tab on the "Play All" Button to play all entries in the chosen playlist to build a playlist queue. 

Why is short tapping on a Video Entry **from a playlist** handles different with the "Next" Video queue?

Cause if i choose a Single Music Video from a playlist -> For Sure i want as next Played Entry the upcoming Video from this chosen playlist and not random stuff while i driving :) - If this not the behaviour (like now) why i can order my playlist entries such easily? It make not much sense ;)  

In Short: Tabbing a Video from a Playlist the Next Video should be the Next Video from the Playlist.
Like this context menu Option -> this should be the Standard for single Tab on the Video (from a Playlist)

See here for demonstration: https://i.imgur.com/2LcFN5S.mp4

**Comment by SameenAhnaf** [comment]
@Delikt Oops! I absolutely forgot to rewrite the solution here. I understand that both issues are different. Anyway, thanks for the reminder.

>A separate setting "Auto-queue constant stream" could be added.
>
>Auto-queue: Preserve YouTube app and website behaviour
Auto-queue constant stream: A single page of enqueued videos (if some videos are manually enqueued) or related videos (if not) will appear and only they will be enqueued. Once streaming all those videos is complete, a new page of related videos will appear.

**Comment by SameenAhnaf** [comment]
@Hogerys What are you even talking about? This issue says nothing about shuffle. Your other issue #6264 asks for that though.

**Comment by Delikt** [comment]
To be sure this Issue will not longer "on hold", because there is the Label "more info needed" isnt this label obsolete now and should be relabeled?

**Comment by Hogerys** [comment]
@SameenAhnaf sorry bro im a mentally aberated autist retard.

**Comment by mhmdanas** [comment]
@Hogerys while I understand that you are self-deprecating, please avoid such language in the future.

**Comment by Hogerys** [comment]
Nah.

**Comment by Delikt** [comment]
why closing this?

**Comment by mhmdanas** [comment]
This user seems to have closed many issues (at least that's what I got in my notifications feed) and then deleted his account.

**Comment by Delikt** [comment]
Should i reopen another issue for this? or a constributer reopen this one?

I really want it happen..

**Comment by mhmdanas** [comment]
No, leave it to me; I'll re-open those issues that I find useful later.
